test/unit/api/test_public_api_objects.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from SeleniumLibrary import SeleniumLibrary
+from SeleniumLibrary.locators import ElementFinder
+from SeleniumLibrary.locators import TableElementFinder
```

Comment:
Last two lines could (and IMHO should) be combined:

    from SeleniumLibrary.locators import ElementFinder, TableElementFinder

Reply:
Done